packs/tfe_fdo_nomad/README.md

```diff
@@ -0,0 +1,162 @@
+| `tfe_redis_use_tls`                          | no       | false                                        | Indicates to use TLS to access Redis.                                                                                                                                                                 |
+| `tfe_redis_use_auth`                         | no       | false                                        | Indicates Redis server is configured to use TFE_REDIS_PASSWORD and TFE_REDIS_USER (optional) for authentication.                                                                                      |
+| `tfe_hostname`                               | yes      | `""`                                         | The host name of the TFE instance to be used while deploying.                                                                                                                                         |
+| `tfe_tls_cert_mount_path`                    | no       | `"/etc/ssl/private/terraform-enterprise"`    | Mount path where the certificates and other files will be mounted inside TFE container.                                                                                                               |
+| `tfe_iact_subnets`                           | no       | `""`                                         | Comma-separated list of subnets in CIDR notation that are allowed to retrieve the initial admin creation token via the API .                                                                          |
+| `tfe_iact_time_limit`                        | no       | `"60"`                                       | Number of minutes that the initial admin creation token can be retrieved via the API after the application starts.                                                                                    |
+| `tfe_vault_disable_mlock`                    | no       | `"false"`                                    | Disable mlock for internal Vault.                                                                                                                                                                     |
+| `tfe_resource_cpu`                           | no       | `"750"`                                      | CPU in MHz for TFE container.                                                                                                                                                                         |
+| `tfe_resource_memory`                        | no       | `"1024"`                                     | Memory in MB for TFE container.                                                                                                                                                                       |
```

Comment:
These variables are of the wrong type:

```suggestion
| `tfe_vault_disable_mlock`                    | no       | `false`                                    | Disable mlock for internal Vault.                                                                                                                                                                     |
| `tfe_resource_cpu`                           | no       | `750`                                      | CPU in MHz for TFE container.                                                                                                                                                                         |
| `tfe_resource_memory`                        | no       | `"1024`                                     | Memory in MB for TFE container.                                                                                                                                                                       |
```

This is just one example of this, but there are _many_ rows of this table that have mismatched types. You should take a pass over the whole table to make sure the types match what's in the `variables.hcl` file.

Reply:
You didn't fix all of these so I've added new comments on each line.